tests/fullstack-test/expr/hex_str.test

```diff
@@ -30,18 +30,14 @@ mysql> alter table test.t set tiflash replica 1;
 func> wait_table test t
 
 mysql> set tidb_enforce_mpp=1; set tidb_isolation_read_engines='tiflash'; select hex(a) from test.t;
-+----------------------------------------------------------------------+
-| hex(a)                                                               |
-+----------------------------------------------------------------------+
-| 7777772E70696E676361702E636F6D                                       |
-| 61626364                                                             |
-| E6B58BE8AF95E6B58BE8AF95E6B58BE8AF95E6B58BE8AF9561626364E6B58BE8AF95 |
-| NULL                                                                 |
-| E38195E38289E381ABE585A5                                             |
-| E6B58BE8AF95E6B58BE8AF95E6B58BE8AF95E6B58BE8AF9561626364E6B58BE8AF95 |
-|                                                                      |
```

Comment:
Why remove this line? By testing in mysql, I found the previous answer is right, could you please check your implement?

Reply:
It seems that our test script is not smart enough and it will filter out those empty string output from mysql-client

https://github.com/pingcap/tiflash/blob/f99b74c393c2ca05d35b7a804200672232026e27/tests/run-test.py#L265-L268

Reply:
same issue here: https://github.com/pingcap/tiflash/pull/5276#issuecomment-1173133426

Reply:
Get, thanks for your information.